lib/commands/general.js

```diff
@@ -25,10 +25,22 @@ commands.doSendKeys = async function (params) {
   return await this.bootstrap.sendAction('setText', params);
 };
 
-commands.getDeviceTime = async function () {
+/**
+ * Get device time
+ * @param {String} format A optional format for `adb shell date`. You can see help with `adb shell date --help`.
```

Comment:
It might be handy to copypaste the part of date command man where possible date format specifiers are described